browser/src/control/Control.JSDialog.js

```diff
@@ -56,6 +56,12 @@ L.Control.JSDialog = L.Control.extend({
 		}
 	},
 
+	closeAll: function() {
+		var dialogs = Object.keys(this.dialogs);
+		for (var i = 0; i < dialogs.length; i++)
+			this.close(dialogs[i]);
```

Comment:
if there will be any pure JS dialog present - we will got 'syntax/parsing error' from the server because we will send message with incorrect window ID which is not expected by the server

Reply:
But I think you can ignore that. Because our pure JS dialogs have custom callback which doesn't send anything to server.